FinSearchUnified/Helper/SnippetExtractor.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace FinSearchUnified\Helper;
+
+use Enlight_View_Default;
+use SimpleXMLElement;
+
+class SnippetExtractor
```

Comment:
This is more like a `QueryInfoMessageParser` than an extractor. But nice restructuring ;)!